### PR TITLE
Testing dev

### PR DESCRIPTION
**Pull request by jnlao**
deleted some code in food-services and restaurant-services that we aren't using and finished all testing but  getDbConnection() in both files

### CODE REVIEW

File: backend/backend.js [hunk 1]
```diff
@@ -68,42 +68,28 @@ app.get("/foods/:id/reviews", async (req, res) => {
...
-    const restName = req.query["name"];
-    try {
-        const result = await restaurantServices.getRestaurants(restName);
-        res.send({ restaurants: result });
-    } catch (error) {
-        console.log(error);
-        res.status(500).send("An error occurred in the server.");
-    }
-});
+// app.get("/restaurants", async (req, res) => {
```
Comment (Daniel-Redfern) [inline backend/backend.js:71]: You can just delete this

File: coverage/lcov-report/restaurant-services.js.html [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+        neighborhood: location,
+    });
+    const validNames = [];
+    for (var i = 0, size = restAtLocation.length; i &lt; size; i++) {
+        validNames.push(restAtLocation[i].name);
+    }
+    return validNames;
+}
+&nbsp;
+// async function findRestByName(rname) {
```
Comment (Daniel-Redfern) [inline coverage/lcov-report/restaurant-services.js.html:261]: If were not using a function you can delete it.

File: coverage/lcov-report/restaurant-services.js.html [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+        <script>
+            window.onload = function () {
+                prettyPrint();
+            };
+        </script>
+        <script src="sorter.js"></script>
+        <script src="block-navigation.js"></script>
+    </body>
+</html>
+
```
Comment (Daniel-Redfern) [inline coverage/lcov-report/restaurant-services.js.html:304]: it seems unhappy that there is no newline at end of file

**Review by Daniel-Redfern — COMMENTED** [review]
Very good use of comments to explain your code, but there are also a few methods that have been commented out which could just be removed.

**Review by AlexBuchko — COMMENTED** [review]
Do we need anything in the coverage folder on the github? Maybe we just want to put it in the git ignore

File: testing/models.test.js [hunk 4]
```diff
@@ -41,11 +48,609 @@ afterAll(async () => {
...
+            price: 1,
+            restaurant_id: "635b7be05455166813f787ad",
+        };
+        let foodResult = new foodModel(dummyFood);
+        await foodResult.save();
+
+        const id = foodResult.id;
+        const rating = "dislikes";
+        const result = await foodServices.incrementFoodRating(id, rating);
+        expect(result).toBeTruthy();
```
Comment (AlexBuchko) [inline testing/models.test.js:193]: Maybe for these increment functions, could have the test being if the likes or dislikes were actually incremented? I'm just worried that he might have issue with this.

**Review by AlexBuchko — APPROVED** [review]

**Review by Ximixiyou — COMMENTED** [review]
for some of the test cases, I am wondering if they are needed, but everything looks good

**Review by Ximixiyou — APPROVED** [review]
looks good

**Pull request merged**